app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreenEffectHandler.kt

```diff
@@ -41,4 +45,19 @@ class BloodPressureHistoryScreenEffectHandler @AssistedInject constructor(
           .map(::BloodPressureHistoryLoaded)
     }
   }
+
+  private fun loadPatient(
+      scheduler: Scheduler
+  ): ObservableTransformer<LoadPatient, BloodPressureHistoryScreenEvent> {
+    return ObservableTransformer { effect ->
+      effect
+          .switchMap {
+            patientRepository
+                .patient(it.patientUuid)
+                .subscribeOn(scheduler)
```

Comment:
I feel maintaining an ongoing subscription to the patient query is unnecessary because the patient details aren't likely to change while the user is on this screen. What do you think of adding a `.take(1)` here after the `.patient()` call to ensure that this patient is only queried once?